src/extension.ts

```diff
@@ -62,6 +62,15 @@ export async function activate(context: vscode.ExtensionContext) {
 	vscode.debug.registerDebugAdapterTrackerFactory(CAMEL_DEBUG_ADAPTER_ID, {
 		createDebugAdapterTracker(session: vscode.DebugSession) {
 		  return {
+			onError: err => {
+				const outputChannel = vscode.window.createOutputChannel("Camel Debug Adapter Protocol");
+				outputChannel.appendLine(err.message);
+				
+				const cp = require('child_process')
+				cp.exec('kamel debug', (_, stdout: string, _) => {
```

Comment:
kamel CLI is unrelated to the Camel Debug Adapter.
`kamel debug` allows to provide a Java debugger connection, not a Camel Debugger.